collector-http_test.go

```diff
@@ -182,38 +182,6 @@ func TestHttpCollector_NonBlockCollect(t *testing.T) {
 
 }
 
-func TestHttpCollector_MaxBatchSize(t *testing.T) {
```

Comment:
There is test for this case: TestHttpCollector_NonBlockCollect, but it did nothing before. I`ve updated ot